depth_to_pointcloud/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>depth_to_pointcloud</name>
+  <version>0.0.0</version>
+  <description>A simple node to convert a depth image and camera info into a point cloud.</description>
```

Comment:
Remark: This code is publishing a PointCloud2, you may want to make it explicit so that we don't think it's a sensor_msgs/PointCloud. This comment can apply to the package name as well if you think it's relevant